modules/virt-vm-creating-nic-web.adoc

```diff
@@ -14,5 +14,6 @@ Create and attach additional NICs to a virtual machine from the web console.
 . Select a virtual machine to open the *Virtual Machine Overview* screen.
 . Click *Network Interfaces* to display the NICs already attached to the virtual machine.
 . Click *Add Network Interface* to create a new slot in the list.
-. Fill in the *Name*, *Model*, *Network*, *Type*, and *MAC Address* for the new NIC.
+. Use the *Network* drop-down menu to select the network attachment definition for the additional network.
```

Comment:
If I recall correctly, we should s/menu/list

Reply:
You do indeed recall correctly :)